data/js/configuring-data.md

```diff
@@ -0,0 +1,164 @@
+# Configuring Data
+
+###### The `api.json` and `api-*.json` files allow us to configure the accessible paths for each data service.
+
+<!-- <article id="undestanding-configuration-files"> -->
+
+## Understanding configuration files
+
+By default WeDeploy Data service is going to use all the JSON files starting with `api-*` and also the file `api.json`.
+
+These files are used to help you manage features such as path validation, authentication, and params validation.
+
+The api JSON files are located at the same path of the `./container.json` and are used following the ordering filesystem.
+
+By the default, if you don't add any api.json file all the collections are going to be exposed.
+
+<!-- </article> -->
+
+<!-- <article id="json-attributes"> -->
+
+## JSON attributes
+
+After understanding how the api configuration files work, it's time to learn what are the supported attributes:
+
+<table class="table">
+  <tr>
+    <th>Field</th> <th>Description</th>
+  </tr>
+  <tr>
+    <td>path</td> <td>The path that represents the collection used to handle the request data.</td>
+  </tr>
+  <tr>
+    <td>data</td> <td>Tells the service if the request to a collection should be stored or not.</td>
+  </tr>
+  <tr>
+    <td>description</td> <td>Used to describe the behavior of an endpoint.</td>
+  </tr>
+  <tr>
+    <td>auth</td> <td>Used to define authentication rules for the collection.</td>
+  </tr>
+  <tr>
+    <td>method</td> <td>HTTP method allowed for the request.</td>
+  </tr>
+  <tr>
+    <td>parameters</td> <td>Params and validation rules for the collection.</td>
+  </tr>
+</table>
+
+**path**
+A path represents the collection used to store your project data.
```

Comment:
Are we not mixing engine concepts with data concepts? I'm just talking about data so far. User should not know about resource path. We're not talking about engine in this case.